v2/examples/example_grpc_tls_server_with_jwt_test.go

```diff
@@ -0,0 +1,32 @@
+package examples_test
+
+import (
+	"context"
+	"net"
+
+	"github.com/spiffe/go-spiffe/v2/grpcworkload"
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	"github.com/spiffe/go-spiffe/v2/spiffejwt"
+)
+
+func Example_gRPCTLSServerWithJWT() {
+	td := spiffeid.TrustDomain("example.org")
+	serverID := spiffeid.Make(td, "server")
+	server, err := grpcworkload.NewTLSServer(context.TODO(),
+		grpcworkload.WithJWTSVIDValidation(serverID.String(), spiffejwt.AllowTrustDomain(td)))
```

Comment:
` AllowTrustDomain` no longer exist you may use `AllowIn` instead?

Reply:
Thanks! Missed this when I renamed stuff.